Automattic/wp-calypso
# Checkout: Fix relative imports in my-sites/checkout

#### Changes proposed in this Pull Request

* Use `eslint --fix` to update all the import statements under checkout

#### Testing instructions

Hopefully if this breaks something, it will break loudly. Check for any TC failures, and also check the changes to make sure only import statements are affected.
